servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/IoUringTest.java

```diff
@@ -52,13 +69,16 @@ void test() throws Exception {
                  BlockingHttpClient client = HttpClients.forSingleAddress(serverHostAndPort(serverContext))
                          .ioExecutor(ioUringExecutor)
                          .buildBlocking()) {
-                HttpResponse response = client.request(client.get(SVC_ECHO));
+                HttpRequest request = client.post(SVC_ECHO).payloadBody("bonjour!", textSerializerUtf8());
+                HttpResponse response = client.request(request);
                 assertThat(response.status(), is(OK));
+                assertThat(response.payloadBody().toString(UTF_8), is("bonjour!"));
```

Comment:
You can use `textSerializerUtf8()` here too :)

Reply:
Done.